apsuite/commissioning_scripts/meas_coupling_tune.py

```diff
@@ -0,0 +1,169 @@
+"""Coupling Measurement from Minimal Tune Separation."""
+
+import numpy as _np
+import matplotlib.pyplot as _plt
+
+
+from ..optimization import SimulAnneal as _SimulAnneal
+
+
+class FitTunes(_SimulAnneal):
```

Comment:
Wouldn't it be better to generalize this class to also perform the measurements?
Or to create a measurement class, in such a way that an object of this class is an attribute of the measurement class?
We already have a `BaseClass` for measurements in the `base.py` module of this subpackage.

Reply:
Yes, I was going to do this (this is why I indicated that the PR is incomplete in the commit message). I wanted to upload this preliminary PR so you all could replicate the coupling fitting I did today.

I will study `base.py`